modules/appeals_api/app/workers/appeals_api/higher_level_review_upload_status_batch.rb

```diff
@@ -27,5 +27,9 @@ def higher_level_review_ids
     def slice_size
       (higher_level_review_ids.length / 5.0).ceil
     end
+
+    def enabled
+      Settings.decision_review.higher_level_review_updater_enabled
```

Comment:
Does this setting actually exist and I guess the better question is why does it exist?

Reply:
it does!
https://github.com/department-of-veterans-affairs/vets-api/blob/master/config/settings.yml#L139

Reply:
and the reason why it does is because this job was pretty much a copy of vba_document's job